interactions/client.py

```diff
@@ -557,6 +557,8 @@ def command(
         options: Optional[
             Union[Dict[str, Any], List[Dict[str, Any]], Option, List[Option]]
         ] = MISSING,
+        name_localizations: Optional[Dict[Union[str, Locale], str]] = MISSING,
```

Comment:
Since this undergoes the same regex as a normal name, it would be convenient if `__check_command` would check for it, too.